service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/NoticeOfProceedingsController.java

```diff
@@ -75,7 +75,8 @@ public AboutToStartOrSubmitCallbackResponse handleMidEvent(@RequestBody Callback
         CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
 
         if (eventValidationService.validateGroup(caseData, NoticeOfProceedingsGroup.class).isEmpty()) {
-            HearingBooking hearingBooking = hearingBookingService.getMostUrgentHearingBooking(caseData);
+            HearingBooking hearingBooking = hearingBookingService
```

Comment:
be nice for this to be extracted into a service... but sure let's get this one closed